### DISCUSSION TITLE
raspbian-ovos - Questions from chats

### DISCUSSION THREAD

**Discussion by builderjer** [OpenVoiceOS/ovos-buildroot#160] [Q&A] [answered]
This was asked in the matrix chats
```
Nice work! ... just a few comments, please don't take them as negative:

    sudo apt update && sudo apt upgrade => should the upgrade add '-y' lest it sit there waiting for you to answer?
    "I have also had better luck installing using pip from a local directory" => shouldn't that be done in a venv?
    "This guide assumes that you are creating a headless device" => Why? I would guess most devices are "headed"
    What about a fuctioning mpc/mpd and pulseaudio playing nice with other audio sources? => this is not an easy task ... I'm still working on a script and some docs to make it easier ... watch this space
    "OVOS-core does not come with any pre-installed skills" => Why? What good is an env't with no skills? How about it comes with one skill? "Hey Mycroft ... install basic skills"
    "There are a few things that are usefull to me. ncpamixer and ovos-cli-client" Why? What do they do?

```

sudo apt upgrade, good call, I will make those changes

I did not do a venv because this was created for a rpi3.  The ovos stack is about all that the rpi3 can handle, so I am assuming that this is the only thing running.  Therefore, there is no need for a venv

Headless device.  Look above.  There are images available for a rpi4 [buildroot](https://drive.google.com/drive/folders/113-zmx6ncoeLNsayseNxoaTlaAk1AfU2) and [manjaro](https://downloads.openvoiceos.com/images/), but these both are for a rpi4.  The rpi3 cannot reliably support a gui

mpc/mpd - This installs a MINIMAL version of ovos.  It is up to the user to expand from there.  An image based on this (coming soon) may change that.

No installed skills - This is by design.  ovos-core is just a library to be used by other software. eg. plugins, skills etc.  Specific distros/images can specify what skills they want installed by default.  This is a minimal install, so it is up to the user to decide what skills are required by the device.

I hope this answers some questions.  ask more please

**Comment by denics** [comment]
Hi @builderjer I agree with almost all your answers :) for the last one:

```
    "There are a few things that are usefull to me. ncpamixer and ovos-cli-client" Why? What do they do?
```

I see the point here. At the moment changing language requires still a lot of work and debugging, for me every new image install and I try to configure in Italian for example, I have to spend a lot of time with the ovos-cli-client to check which skill / plugin is missing the translation, which utterance is not understood (most of the translations are still machine translations) etc.
I upvote ovos-cli-client :D

**Reply by builderjer** [reply]
both ncpamixer and ovos-cli-client are both installed on the image.  I am working on a better one now

**Comment by denics** [comment]
up to now, all images, buildroots, manjaro, are provided in English, which is perfect. However, we know that our beloved Ovos can support many other languages. My request is: can we have a skill that at the first install ask, in different langages, why not depending on location, to set the language ?

**Reply by builderjer** [reply, marked as answer]
I would like to move this here
https://github.com/orgs/OpenVoiceOS/discussions/163#discussion-5037089
